actors/evm/src/interpreter/instructions/state.rs

```diff
@@ -57,20 +57,21 @@ mod test {
                 (false, U256::MAX),
             ] {
                 evm_unit_test! {
-                    (rt, m) {
+                    (rt:
+                        rt.in_call = true;
```

Comment:
this just looks oit of place...
Lets keep `(rt) {init...}` which looks like a closure instead of rt being a label.

I also like `{}` around m code as it was more.